frigate/video.py

```diff
@@ -190,6 +190,8 @@ def track_camera(name, config, global_objects_config, frame_queue, frame_shape,
     # load in the mask for object detection
     if 'mask' in config:
         mask = cv2.imread("/config/{}".format(config['mask']), cv2.IMREAD_GRAYSCALE)
+        if mask.size == 0:
+            print("Failed to read mask file")
```

Comment:
Probably better to set `mask = None` here so frigate doesn't error out later.

Reply:
Good catch!